src/Android/Avalonia.Android/Platform/SkiaPlatform/TopLevelImpl.cs

```diff
@@ -39,8 +39,10 @@ public TopLevelImpl(Context context, bool placeOnTop = false)
             _gl = GlPlatformSurface.TryCreate(this);
             _framebuffer = new FramebufferManager(this);
 
-            MaxClientSize = new Size(_view.Resources.DisplayMetrics.WidthPixels,
-                _view.Resources.DisplayMetrics.HeightPixels);
+            RenderScaling = (int)_view.Resources.DisplayMetrics.Density;
```

Comment:
I wasn't sure about this, because `Density` uses `DensityDefault` (= 160) as base, and if I'm not mistaken `RenderScaling` uses 96 as base. But I tried using `(int)DensityDpi / 96.0` and everything looked too big, so I'm not sure what's happening.

Reply:
The whole 96 thing is a leftover from the initial win32-centric implementation. The idea was to just use a scaling multiplier instead of "DPI that is not quite DPI" metric